scripts/setup-centos.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+TARGET=$1
+
+# TODO: Assert TARGET is one of "el6" and "el7"
+
+echo "[Install] dependencies"
+sudo yum update
+
+sudo yum install -y linux-image-extra-$(uname -r)
+sudo yum install -y git curl wget
+
+echo "[Install] docker-compose"
+if [ "$TARGET" = "el6" ]; then
+  # el6 does not work with docker-compose 1.4+
```

Comment:
Alternative approach would be to build Packages on Ubuntu14 node and spin up respective Trusty/Xenial/EL6/EL7 Vagrant node to install just produced packages by docker-compose (if needed).